WcaOnRails/spec/factories/competitions.rb

```diff
@@ -76,5 +76,11 @@
       with_delegate
       showAtAll true
     end
+
+    trait :entry_fee do
+      base_entry_fee_lowest_denomination 1000
+      currency_code "AUD"
+      connected_stripe_account_id "acct_19ZQVmE2qoiROdto"
```

Comment:
@jfly - I emailed Stripe support and they inform me that if the WCA uses their live keys, then it will only connect to a live account so a user cannot connect with their test keys to our live keys.

From Stripe support:

> SInce the platform is using the Live API Key, the platform will treat any information passed along the API as live data. If test details are being entered through your onboarding process for the platform, your user's should see an error saying that they are entering known test details. Ultimately, in this scenario a account couldn't connect with this information and the Dashboard being in test/live mode won't have an effect.